src/main/java/uk/gov/laa/ccms/caab/client/EbsApiClient.java

```diff
@@ -210,5 +254,135 @@ public Mono<UserDetails> getUsers(final Integer providerId) {
 
   }
 
+  /**
+   * Retrieves proceeding detail.
+   *
+   * @return A Mono containing the ProceedingDetail or an error handler if an error occurs.
+   */
+  public Mono<ProceedingDetail> getProceeding(String proceedingCode) {
+    return ebsApiWebClient
+        .get()
+        .uri("/proceedings/{proceeding-code}", proceedingCode)
+        .retrieve()
+        .bodyToMono(ProceedingDetail.class)
+        .onErrorResume(e -> ebsApiClientErrorHandler.handleProceedingError(proceedingCode, e));
+  }
+
+  /**
+   * Retrieves scope limitation details which match the provided example ScopeLimitationDetail.
+   *
+   * @return A Mono containing the ScopeLimitationDetails or an error handler if an error occurs.
+   */
+  public Mono<ScopeLimitationDetails> getScopeLimitations(
+      ScopeLimitationDetail scopeLimitationDetail) {
```

Comment:
missing @param in javadoc
final on params

Reply:
done